src/manager/manager.c

```diff
@@ -814,6 +834,63 @@ static int manager_name_owner_changed(sd_bus_message *m, void *userdata, UNUSED
         return 0;
 }
 
+static int manager_reset_debug_monitor_timer(Manager *manager, sd_event_source **event_source);
+
+static int manager_debug_monitor_timer_callback(
+                sd_event_source *event_source, UNUSED uint64_t usec, void *userdata) {
+        Manager *manager = (Manager *) userdata;
+
+        /* List connected nodes. */
+        Node *node = NULL;
+        LIST_FOREACH(nodes, node, manager->nodes) {
+                if (node->agent_bus != NULL) {
+                        hirte_log_debugf(
+                                        "%s:%d: %s(): connected node name = %s",
+                                        __FILE__,
+                                        __LINE__,
+                                        __func__,
+                                        node->name);
```

Comment:
```suggestion
                        hirte_log_debugf("connected node name = %s", node->name);
```
`hirte_log_debug` already adds the file, line and func info.

Reply:
I find a periodic function useful as a quick way to add a print and observe the effect of some event.  Sure, sometimes a debugger is needed.